modules/gdscript/gdscript.cpp

```diff
@@ -478,7 +478,7 @@ void GDScript::_clear_doc() {
 void GDScript::_update_doc() {
 	_clear_doc();
 
-	doc.script_path = "\"" + get_path().get_slice("://", 1) + "\"";
+	doc.script_path = "\"" + get_script_path().get_slice("://", 1) + "\"";
```

Comment:
You could take the opportunity to lean out the code using `vformat()` for strings.

Reply:
good point!